shared/config/constants.json

```diff
@@ -0,0 +1,31 @@
+{
+  "enums": [{
+    "comment": "Used to communicate different types of levels",
+    "contents": {
+      "LevelKind": {
+        "peer_review": "peer_review",
+        "assessment": "assessment",
+        "puzzle": "puzzle",
+        "unplugged": "unplugged",
+        "level": "level"
+      }
+    }
+  },
+
+  {
+    "comment": "Different possibilities for level.status, used to communicate how user has performed on a given level",
```

Comment:
We could instead have the source of truth be a ruby (or js) module instead of a JSON file, with formatting, comments, etc. and automatically generate the javascript counterpart as part of the build process.